elements/pfe-styles/src/pfe-typography-classes.scss

```
@@ -1,14 +1,15 @@
 @import "../../pfe-sass/pfe-sass";
 
 // TITLES
-@each $size in (6xl, 5xl, 4xl, 3xl, 2xl, xl, lg, md, sm, xs) {
+@each $size in ("6xl", "5xl", "4xl", "3xl", "2xl", "xl", "lg", "md", "sm", "xs") {
   .pfe-title--#{$size}  {
-      @extend %pfe-title--#{$size}; 
+      @include pfe-title($sizing: #{$size}, $important: true); 
   }
 }
+
 // TEXT
-@each $size in (lg, md, sm, xs) {
-  .pfe-content--text-#{$size}  {
-      @extend %pfe-content--text-#{$size}; 
+@each $size in ("xl", "lg", "md", "sm", "xs") {
```

```suggestion
@each $size in ("xl", "lg", "md", "sm", "xs") {
```

No quotes necessary here

---

do you think it would be difficult for devs to remember when to use quotes vs. not using them depending on the value they are passing in? When I was testing with webrh, the mixins seemed to fail without quotes. Idk if it was a fluke but it worked with them.

---

It's just extra typing when you don't need it.  You only need it when starting the string with a number because it gets confused and thinks you're using a numerical type instead of a string type.